internal/p2p/router.go

```diff
@@ -444,8 +444,8 @@ func (r *Router) filterPeersID(ctx context.Context, id types.NodeID) error {
 func (r *Router) dialSleep(ctx context.Context) {
 	if r.options.DialSleep == nil {
 		const (
-			maxDialerInterval = 3000
-			minDialerInterval = 250
+			maxDialerInterval = 500
+			minDialerInterval = 100
```

Comment:
Could 100ms be too low. What are the consequences if this value is too low?

Reply:
I'm not super worried about this. @creachadair suggested setting min to 3ms and max to 100ms so I felt like I was being conservative. 

I think it means we dial specific peers more regularly if they fail.

Reply:
yeah, I think it's cheap enough that it's safe to do it more often, and think we should degrade the dialPeer error logging but I think it's good to know that we're *trying* to dial peers, so maybe info rather than debug?

Reply:
I reduced the transient (connection timed out, etc.) logging to debug.